src/instructlab/model/train.py

```diff
@@ -393,6 +388,11 @@ def clickpath_setup(is_dir: bool) -> click.Path:
     is_flag=True,
     help="By default, checkpoints are saved at the end of each training epoch. This option disables this behavior.",
 )
+@click.option(
+    "--pipeline",
+    type=click.Choice(["simple", "full", "accelerated"]),
```

Comment:
Add help text

Reply:
Yes - how do I know which to choose for my hardware? What are the tradeoffs?  Why do we default to simple?